problems/[221215] 1328. Break a Palindrome.js

```diff
@@ -5,5 +5,26 @@
  * @return {string}
  */
 const breakPalindrome = function (palindrome) {
+  if (palindrome.length === 1) return "";
 
+  const result = palindrome.replace(/[^a]/, "a");
+
+  const deduplicate = new Set(result.split(""));
```

Comment:
```suggestion
  const deduplicate = new Set(result);
```

new Set()은 첫 번째 파라미터로 iterable을 받아 Set 객체로 만들어주는 역할을 해주는 생성자 함수인데요, `result` 자체가 문자열(String)이므로 iterable입니다.

굳이 split()을 해주지 않아도 됩니다~

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/Set

Reply:
오오 그렇네요!! 👍